### ISSUE TITLE
Schalterwippe - Stopp funktioniert nicht mehr

### ISSUE THREAD

**Issue by Bopp77** [Zefau/ioBroker.jarvis#829]
*Please note that you may restore an old layout via jarvis adapter configuration directly within ioBroker

**Prerequisites / Voraussetzungen**
- [ ] Which jarvis version are you running / Welche Version von jarvis nutzt du? 
- [ ] VERSION 2.2.1
- [ ] 
- [ ] Fehler / Screenshot der Browser Konsole (F12), siehe [Fehleranalyse und -Meldung](https://github.com/Zefau/ioBroker.jarvis/wiki/de-Errors)

**Describe the bug / Beschreibe den Fehler**
Ich steuere meine Jalousien mit Shelly-Rolladen WLAN-Aktoren.
Seit dem Update von Jarvis fehlt nach dem Laden der Oberfläche die Stopp-Schaltfläche der Jarvis-Schalterwippe. Nach Betätigen von Hoch oder runter ergänzt sich die Stopp-Schaltfläche zwar, ein stoppen ist aber weder über diese noch über ein erneutes Drücken der selben Richtung mehr möglich.
Die Jarvis-Schalter funktionierten vor dem Update super. 
Über das Shelly-eigene Webinterface ist auch aktuell alles normal ansteuerbar. 

![Screenshot_20210709-103527_Chrome](https://user-images.githubusercontent.com/87174804/125050935-bde29900-e0a2-11eb-8e4e-04895ebb8c81.jpg)

**Comment by mcuiobroker** [comment]
Schau Dir mal die Links an:
https://forum.iobroker.net/topic/30668/jarvis-v2-1-0-just-another-remarkable-vis/3568 und ff
https://forum.iobroker.net/topic/30668/jarvis-v2-1-0-just-another-remarkable-vis/3577 und ff
https://forum.iobroker.net/topic/30668/jarvis-v2-2-0-just-another-remarkable-vis/3115
Vielleicht hilft es bei den Einstellungen, ansonsten bitte im Forum mit allen Einstellungen vom Gerät noch mal das Problem ansprechen.

**Comment by realhawker** [comment]
Ich habe das gleiche Problem und habe alle Einstellungen so wie in den Links angeben/beschrieben. Vor dem letzten Update hat es auch noch funktioniert. Gibt es noch weitere Dinge die ich probieren kann um das Problem zu beheben/einzugrenzen?

**Comment by Zefau** [comment]
Wie sieht es bei v3 aus?

**Comment by realhawker** [comment]
Hab das Update auf v3 gemacht. Dort klappt es leider auch nicht. Kann ich irgendwie Logs abziehen?

**Comment by Zefau** [comment]
Also er zeigt zwei Pfeile an, aber nicht den Stop-Button? Sowohl im Geräte-Popup als auch im Layout?

Wie ist das Gerät konfiguriert? Und wie ist insbesondere der BlindLevelAction konfiguriert?

**Comment by realhawker** [comment]
Doch, der Stop-Button wird angezeigt, aber er funktioniert nicht.

**Comment by Zefau** [comment]
> Wie ist das Gerät konfiguriert? Und wie ist insbesondere der BlindLevelAction konfiguriert?

**Comment by realhawker** [comment]
Bisher hatte es immer funktioniert, ohne das ich was spezielles konfiguriert habe. Aber ich habe gerade mal schaut was konfiguriert ist (falls ich den richtigen Dialog gefunden habe). Es scheint nichts konfiguriert zu sein. Jedoch ist es im Dialog nicht möglich etwas auszuwählen: 
![grafik](https://user-images.githubusercontent.com/4715835/128402749-2046d26d-6238-4f7a-b714-9c904c0d94a8.png)
Muss ich was spezielles tun um dort etwas einstellen zu können?

**Comment by Zefau** [comment]
du bist auf `v3.0.0-alpha.47` ?

**Comment by realhawker** [comment]
Ja
![grafik](https://user-images.githubusercontent.com/4715835/128403318-66962fab-fe23-4f35-846b-2f32a1ab7eff.png)

**Comment by Zefau** [comment]
Hat das Gerät keine weiteren Datenpunkte?

**Comment by Zefau** [comment]
Also du musst für den Stop-Button die entsprechenden Datenpunkte auswählen. Dafür ist das Dropdown.

So sieht es bei mir aus:

![image](https://user-images.githubusercontent.com/14368811/128403771-06251bf6-c450-4c70-9b58-4655a51ccf43.png)

Gerät:

![image](https://user-images.githubusercontent.com/14368811/128403847-f2918b97-d889-4ffb-b044-be5d1c7121ea.png)

State:

![image](https://user-images.githubusercontent.com/14368811/128403873-5cad5a0e-b4c8-48ab-8874-8752f23aa146.png)

**Comment by realhawker** [comment]
Es gibt weitere Datenpunkte
![grafik](https://user-images.githubusercontent.com/4715835/128403816-0dbec384-0a6f-49b8-ab34-a6d72e71c38a.png)

**Comment by realhawker** [comment]
Hier scheint was schief zu laufen, oder sehe ich das falsch?
![grafik](https://user-images.githubusercontent.com/4715835/128404667-05b24979-fc02-4ded-b0e0-951fe543f4e9.png)

**Comment by Zefau** [comment]
Schau mal bitte in der Browser-Konsole (via F12)

**Comment by realhawker** [comment]
![grafik](https://user-images.githubusercontent.com/4715835/128405148-a514c372-e1d9-4fc1-97a1-43937ebeffa7.png)

**Comment by Zefau** [comment]
Sieht korrekt aus. Der Fehler hängt mit Icons zusammen und kann ignoriert werden. Keine Ahnung, warum du die Datenpunkte nicht im Dropdown angezeigt bekommst. Sehr merkwürdig.

**Comment by Zefau** [comment]
Hab den Fehler gefunden. Ich fixe. Gibt gleich ein Update.

**Comment by realhawker** [comment]
Perfekt. Vielen Dank.

**Comment by Zefau** [comment]
probier mal `v3.0.0-alpha.48`

**Comment by realhawker** [comment]
Da scheint beim Update jetzt richtig was schief gelaufen zu sein.
![grafik](https://user-images.githubusercontent.com/4715835/128409275-faff87b5-b23b-47e2-93dc-9241e051c670.png)

**Comment by Zefau** [comment]
sorry, probier gleich nochmal `v3.0.0-alpha.49`.

**Comment by realhawker** [comment]
Jetzt funktioniert es. Ich nehme mal an, dass meine Konfiguration durch das Update gelöscht wurde, oder kann ich das noch irgendwie wieder herstellen?

**Comment by realhawker** [comment]
Hab noch ein Backup wieder herstellen können und hab scheinbar nur ein paar wenige Änderungen verloren. Also kein Problem. Danke, dass du dich um das Problem gekümmert hast.
